README.md

```diff
@@ -1 +1,1090 @@
-# PRE DRAFT repository for a coding style PER
+# Coding Style Guide
```

Comment:
Personally I'd like to separate the spec from the README. For me the README should be a quick introduction to the repo with pointers where to find more information. The spec should IMHO be a separate document.

Reply:
Done.